Split/Storage/Impressions/ImpressionDao.swift

```diff
@@ -86,6 +86,9 @@ class CoreDataImpressionDao: BaseCoreDataDao, ImpressionDao {
     }
 
     func delete(_ impressions: [Impression]) {
+        if impressions.count == 0 {
```

Comment:
shouldn't this check be done also on the EventsDao::delete method ?

Reply:
Yup. Added doc. Thank you!